src/config/mod.rs

```diff
@@ -3206,6 +3211,46 @@ impl ConfigManager for LogConfigManager {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug, OnlineConfig)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct MemoryConfig {
+    // Whether enable the heap profiling which may have a bit performance overhead about 2% for the
+    // default sample rate.
+    #[online_config(skip)]
+    pub enable_heap_profiling: bool,
```

Comment:
Seems it's not hard to support online config, why disable it.

Reply:
I have no time to support it and test it before the code freeze.

Reply:
let me try to add it this afternoon.